force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -0,0 +1,12 @@
+<template>
```

Comment:
Let's apply scrollable y to both of these, move the height 100% over to a css class using the namespace-attribut_effect format (like eda-height_full), apply that to the grid and columns, remove the template container because it's not documented, and apply slds-no-flex to the columns to make certain they stay in place.